### PR TITLE
Better document FE_Nothing.

### PR DESCRIPTION
**Pull request by bangerth**
/rebuild

### CODE REVIEW

**Review by masterleinad — APPROVED** [review]
Looks reasonable to me.

File: include/deal.II/fe/fe_nothing.h [hunk 1]
```diff
@@ -41,6 +41,57 @@ DEAL_II_NAMESPACE_OPEN
  * use cases for this element. An interesting application for this element is
  * also presented in the paper @cite Cangiani2012.
  *
+ *
+ * <h3>FE_Nothing as seen as a function space</h3>
+ *
+ * Finite elements are often best interpreted as forming a
+ * [function s[ace](https://en.wikipedia.org/wiki/Function_space), i.e., a
```
Comment (marcfehling) [inline include/deal.II/fe/fe_nothing.h:48]:
```suggestion
 * [function space](https://en.wikipedia.org/wiki/Function_space), i.e., a
```

File: include/deal.II/fe/fe_nothing.h [hunk 2]
```diff
@@ -83,15 +134,16 @@ class FE_Nothing : public FiniteElement<dim, spacedim>
    *
    * The first argument specifies the reference-cell type.
    *
-   * The second argument denotes the number of components to give this
-   * finite element (default = 1).
+   * Constructor. The second argument, `n_components`, denotes the number of
```
Comment (marcfehling) [inline include/deal.II/fe/fe_nothing.h:137]:
```suggestion
   * The second argument, `n_components`, denotes the number of
```
Line 133 already mentions that this is a constructor.

**Review by marcfehling — COMMENTED** [review]
Just minor suggestions.

**Comment by bangerth** [comment]
Thanks, @marcfehling ! I've fixed these points, and also used proper doxygen markup for arguments.

**Review by marcfehling — APPROVED** [review]

**Review by kronbichler — APPROVED** [review]

**Pull request merged**